docs/trestle_author.md

```diff
@@ -223,3 +223,91 @@ trestle_root
 #### Recursive (`-r`, `--recurse`)
 
 By default `author headers` only indexes a flat directory. The recursive option allows the discovery of sub directories.
+
+## `trestle author ssp`
+
+The `ssp` sub-command creates a partial ssp from a catalog, profile and yaml header file.  It can also assemble the markdown files
+into a single json SSP file.  The catalog consists of a number of controls with parameters, and the profile specifies a subset of
+those controls along with corresponding parameter values.  This command merges the information from the two files and generates a
+directory containing a set of markdown files, one for each control in the profile.  Each markdown file has the yaml header embedded
+at the start of the file.
+
+Example usage for creation of the markdown:
+
+`trestle author ssp -f my_prof -yh /my_yaml_dir/header.yaml -s "ImplGuidance:Implied Guidance,ExpectedEvidence:Expected Evidence" -o my_ssp setup`
+
+In this example the catalog and profile have previously been imported into the trestle project directory, making sure to import the
+catalog using the name specified in the import href of the profile.  Note that the path in the href is ignored.  The yaml
+header file can be anywhere in the file system.  In this case the two sections loaded are `ImplGuidance` and `ExpectedEvidence` -
+and their aliases are provided with full spacing and spelling so the section headers will have proper titles.  The output will be
+placed in the trestle subdirectory `my_ssp` with a subdirectory for each control group.
+
+<br>
+<details>
+<summary>The resulting files look like this</summary>
+
+```
+---
+control-origination:
+- Service Provider Corporate
+- Service Provider System Specific
+responsible-roles:
+- Customer
+---
+
+
+# ac-1 - AC Access Control Policy and Procedures
+## Control Description
+
+The organization:
+
+a. Develops, documents, and disseminates to Assignment: organization-defined personnel or roles:
+
+    1. An access control policy that addresses purpose, scope, roles, responsibilities, management commitment, coordination among organizational entities, and compliance; and
+    2. Procedures to facilitate the implementation of the access control policy and associated access controls; and
+
+b. Reviews and updates the current:
+
+    1. Access control policy at least annually; and
+    2. Access control procedures at least annually.
+
+
+
+## ac-1 What is the solution and how is it implemented?
+
+---
+### Part a.
+
+Add control implementation description here.
+
+
+---
+### Part b.
+
+Add control implementation description here.
+
+---
+
+
+## ac-1 Section Implied Guidance
```

Comment:
I think this line is meant to be implementation guidance.

Reply:
fixed